cmd/kola/options.go

```diff
@@ -50,3 +57,24 @@ func init() {
 	sv(&kola.AWSOptions.InstanceType, "aws-type", "t1.micro", "AWS instance type")
 	sv(&kola.AWSOptions.SecurityGroup, "aws-sg", "kola", "AWS security group name")
 }
+
+// Sync up the command line options if there is dependency
+func syncOptions() {
+	// check if target board is default or not
+	if kola.QEMUOptions.Board != defaultTargetBoard {
+		// Is the target board supported?
+		if kola.QEMUOptions.Board != "arm64-usr" {
```

Comment:
instead of checking for arm64-usr, just try to index `kolaDefaultImages`.

also, i dont think you should override user's settings here - just error out of the board isn't in `kolaDefaultImages`.